WordPress/Classes/ViewRelated/NUX/LoginEpilogueUserInfo.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+/// A simple container for the user info shown on the login epilogue screen.
+///
+struct LoginEpilogueUserInfo {
+    var username = ""
+    var fullname = ""
+    var email = ""
+    var gravatarUrl: String?
+
+    init(account: WPAccount) {
+        if let name = account.username {
+            username = "@\(name)"
```

Comment:
We end up with double `@`s, because it's being added both here and in `LoginEpilogueUserInfoCell.swift`. IMO leave it there and remove it here, since it seems like a touch added for presentation.